pkg/kotsutil/kots.go

```diff
@@ -273,7 +273,7 @@ func (o KotsKinds) Marshal(g string, v string, k string) (string, error) {
 					return "", errors.Wrap(err, "failed to encode identityconfig")
 				}
 				return string(b.Bytes()), nil
-			case "HelmChart":
+			case "HelmChartList":
```

Comment:
Changed this since we're actually marshalling a `HelmChartList` when this is called.